libraries/libstratosphere/include/stratosphere/fs/fs_read_only_filesystem.hpp

```diff
@@ -115,57 +115,57 @@ namespace ams::fs {
+                return fs::ResultUnsupportedWriteForReadOnlyFileSystem();
             }
 
             virtual Result DoCreateDirectory(const fs::Path &path) override final {
                 AMS_UNUSED(path);
-                return fs::ResultUnsupportedOperationInReadOnlyFileSystemTemplateA();
+                return fs::ResultUnsupportedWriteForReadOnlyFileSystem();
             }
 
             virtual Result DoDeleteDirectory(const fs::Path &path) override final {
                 AMS_UNUSED(path);
-                return fs::ResultUnsupportedOperationInReadOnlyFileSystemTemplateA();
+                return fs::ResultUnsupportedWriteForReadOnlyFileSystem();
             }
 
             virtual Result DoDeleteDirectoryRecursively(const fs::Path &path) override final {
                 AMS_UNUSED(path);
-                return fs::ResultUnsupportedOperationInReadOnlyFileSystemTemplateA();
+                return fs::ResultUnsupportedWriteForReadOnlyFileSystem();
             }
 
             virtual Result DoRenameFile(const fs::Path &old_path, const fs::Path &new_path) override final {
                 AMS_UNUSED(old_path, new_path);
-                return fs::ResultUnsupportedOperationInReadOnlyFileSystemTemplateA();
+                return fs::ResultUnsupportedWriteForReadOnlyFileSystem();
             }
 
             virtual Result DoRenameDirectory(const fs::Path &old_path, const fs::Path &new_path) override final {
                 AMS_UNUSED(old_path, new_path);
-                return fs::ResultUnsupportedOperationInReadOnlyFileSystemTemplateA();
+                return fs::ResultUnsupportedWriteForReadOnlyFileSystem();
             }
 
             virtual Result DoCleanDirectoryRecursively(const fs::Path &path) override final {
                 AMS_UNUSED(path);
-                return fs::ResultUnsupportedOperationInReadOnlyFileSystemTemplateA();
+                return fs::ResultUnsupportedWriteForReadOnlyFileSystem();
             }
 
             virtual Result DoGetFreeSpaceSize(s64 *out, const fs::Path &path) override final {
                 AMS_UNUSED(out, path);
-                return fs::ResultUnsupportedOperationInReadOnlyFileSystemTemplateB();
+                return fs::ResultUnsupportedCommitProvisionallyForReadOnlyFileSystem();
             }
 
             virtual Result DoGetTotalSpaceSize(s64 *out, const fs::Path &path) override final {
                 AMS_UNUSED(out, path);
-                return fs::ResultUnsupportedOperationInReadOnlyFileSystemTemplateB();
+                return fs::ResultUnsupportedCommitProvisionallyForReadOnlyFileSystem();
             }
 
             virtual Result DoCommitProvisionally(s64 counter) override final {
                 AMS_UNUSED(counter);
-                return fs::ResultUnsupportedOperationInReadOnlyFileSystemTemplateC();
+                return fs::ResultUnsupportedGetTotalSpaceSizeForReadOnlyFileSystem();
```

Comment:
In case it was missed, these are reversed.

Reply:
I'll need to check if these are wrong names or if these are wrong values.